android/MobiledgeXSDKDemo/matchingenginehelper/src/main/java/com/mobiledgex/matchingenginehelper/MatchingEngineHelper.java

```diff
@@ -311,23 +311,15 @@ public void startEdgeEvents(boolean newConnection) {
             }
 
             if (mEdgeEventsRunning) {

```

Comment:
@lgarner this method is much simplified now, but seems to be working.

Reply:
nice refactoring